content/maxwell3_fdem/MT/MT_N_layered_Earth.rst

```diff
@@ -241,11 +241,64 @@ These coefficients tell us how much energy of the incoming has been reflected or
 Refraction angle
 ****************
 
+.. figure:: images/RefractionAngle.png
+ :align: right
+ :scale: 50% 
+ :name: Refraction_MT
+
+ Refraction and Reflection angles
+
+
 In reality, the incident wave is coming from all the possible directions in the air. So how valid is our assumption of an incident vertical wave?
 
 What is important is the refraction angle at the Air-Earth interface, the angle of the transmitted wave in the ground.
 
-As any wave, electromagnetic waves follow Snell's law
+As any wave, electromagnetic waves follow Snell's law, that we can derive from the Maxwell's equation.
```

Comment:
could link Snell's law to wikipedia if you like, and perhaps Maxwell's equations to `maxwells_equations_index`